source/modulo_components/modulo_components/component_interface.py

```diff
@@ -330,6 +330,18 @@ def trigger(self, trigger_name: str):
         self._triggers[trigger_name] = True
         self._publish_predicate(trigger_name)
 
+    def remove_output(self, signal_name):
+        if signal_name not in self._outputs.keys():
+            parsed_signal_name = parse_topic_name(signal_name)
+            if parsed_signal_name not in self._outputs.keys():
+                self.get_logger().debug(f"Unknown output '{signal_name}' (parsed name was '{parsed_signal_name}').")
+                return
+            signal_name = parsed_signal_name
+        if "publisher" in self._outputs[signal_name]:
```

Comment:
Is this correct or should it be `self._outputs[signal_name].keys()`?

Reply:
no that is not correct :+1: